keras/layers/preprocessing/benchmarks/image_preproc_benchmark.py

```diff
@@ -86,7 +86,7 @@ def run_dataset_implementation(self, batch_size):
       ds = ds.prefetch(batch_size)
       img_augmentation = functools.partial(
           image_augmentation, batch_size=batch_size)
-      ds = ds.map(img_augmentation)
+      ds = ds.map(img_augmentation,num_parallel_calls=tf.data.experimental.AUTOTUNE)
```

Comment:
We may want to hard code a value (like `8`) here instead of using autotune, to guarantee determinism and reproducibility.